src/components/common/Toggle/index.js

```diff
@@ -0,0 +1,31 @@
+import React  from "react";
+import PropTypes from "prop-types";
+import * as Layout from "./style.module.css";
+import { useState } from "react";
+import {IoIosArrowUp, IoIosArrowDown} from "react-icons/io";
+
+const Toggle =(props)=>{
+	const [showDiv, setvisiblility] = useState(true);
```

Comment:
The current toggle will always be open . I need a param through which I can decide from parent to show it or hide it